DataCleaningScripts/Add_months_PortalPlots.R

```diff
@@ -1,5 +1,56 @@
+#' Appends new dates to Portal_plots
+#'
+#'
+#' 
+#' @example update_portal_plots()
+#' 
+#'
+#'
+library(zoo)
+update_portal_plots <- function() {
+#load plot data
+portal_plots = read.csv("../SiteandMethods/Portal_plots.csv")
+# load rodent data
+rodentdat = read.csv("../Rodents/Portal_rodent.csv",stringsAsFactors = F,as.is=T,na.strings = '')  
+
+# define currrent plot treatments
+
+controls = c(4,5,6,7,11,13,14,17,18,24)
+removals = c(1,9,10,12,16,23)
+exclosures = c(2,3,8,15,19,20,21,22)
+
+# proceed only if rodentdat has more recent data than plot data
+lastplotdates=max(as.yearmon(paste(substr(portal_plots$yr,3,4),portal_plots$month), "%y%m"))
+newdates=max(as.yearmon(paste(substr(rodentdat$yr,3,4),rodentdat$mo), "%y%m"))
```

Comment:
Addressing the issues of having more than one month of data added at a time. What about generating a vector of the `yearmon` combinations that exist in the plots table, and a vector of the `yearmon` combinations that exist in the rodents table, and then creating a vector of the ones that aren't in plots and adding new values for each of them.

Basic logic would look like:

```
plots = c(1, 2, 3, 4)
rodents = c(1, 2, 3, 4, 5, 6)
missing_dates = setdiff(rodents, plots)
missing_modern_dates = missing_dates[missing_dates > max(plots)] #check to avoid adding old data with with incorrect treatments
for (date in missing_modern_dates){
    do stuff
}
```